front/app/components/admin/ResourceList/SortableList.tsx

```diff
@@ -64,20 +59,16 @@ const SortableList = ({
 
   const handleDragRow = (fromIndex: number, toIndex: number) => {
     const listItems = getListItems();
-    if (!listItems) return;
```

Comment:
`listItems` is always an array, so this if condition is always `false` & `return` is never reached.